netbox_interface_name_rules/templates/netbox_interface_name_rules/buttons/export_yaml_only.html

```
@@ -0,0 +1,33 @@
+  <button type="button" class="btn btn-purple dropdown-toggle" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+    <i class="mdi mdi-download" aria-hidden="true"></i> {{ label }}
+  </button>
+  <ul class="dropdown-menu dropdown-menu-end">
+    <li><a class="dropdown-item" href="?{% if url_params %}{{ url_params }}&{% endif %}export">{% trans "All Data" %} ({{ data_format }})</a></li>
+    {% if export_templates %}
+      <li>
+        <hr class="dropdown-divider">
+      </li>
+      {% for et in export_templates %}
+        <li>
+          <a class="dropdown-item" href="?{% if url_params %}{{ url_params }}&{% endif %}export={{ et.name }}"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)